index.js

```diff
@@ -128,6 +146,9 @@ function magnetURIEncode (obj) {
         if (key === 'kt') val = encodeURIComponent(val)
         if (key === 'so') return
 
+        // translate back pe into x.pe
+        if (key === 'pe') key = 'x.pe'
```

Comment:
Rather than transforming the key to a temporary `'pe'` value so that it can make it through the `filter` function, I'd rather add an exception to the filter to allow any 2 length key or `'x.pe'` through. I think having one exception in the code is clearer than having two places in the code to handle this exception -- one where it's transformed and then later where it is un-transformed.